src/main/webapp/app/overview/course-exams/course-exams.component.html

```diff
@@ -28,7 +28,7 @@ <h5 jhiTranslate="artemisApp.exam.overview.testExamsHeading">Test Exams</h5>
                     <jhi-course-exam-detail
                         [exam]="exam"
                         [course]="course!"
-                        [maxAttemptsReached]="getStudentExamForExamIdOrderedByIdReverse(exam!.id!).length > 0"
+                        [maxAttemptsReached]="getStudentExamForExamIdOrderedByIdReverse(exam!.id!).length === 255"
```

Comment:
Is this < 255 check only happening in the front end? I guess if we want to make sure we should also check this in the back end.
Also I'd change the === to >=, just in case

Reply:
yes, this is ensured in the database using index